lib/ansible/modules/cloud/amazon/aws_s3_bucket_info.py

```diff
@@ -32,7 +34,11 @@
 # Note: Only AWS S3 is currently supported
 
 # Lists all s3 buckets
-- aws_s3_bucket_facts:
+- aws_s3_bucket_info:
+  register: result
+
+- name: List buckes
```

Comment:
```suggestion
- name: List buckets
```

Reply:
Good spot! Thanks.

You mean the 2019/08/08 meeting, right? Do you know who'll chair it?

Reply:
Yes, thanks - calendars are hard.  :)  I do not, it's before my start of day so I'm just generally not around for the Thursday meetings.